{"msg": "virtual machine centos71_0 not found", "exception": "Traceback (most recent call last):\n  File \"/tmp/ansible_virt_payload_iiza19r9/ansible_virt_payload.zip/ansible/modules/cloud/misc/virt.py\", line 593, in main\n  File \"/tmp/ansible_virt_payload_iiza19r9/ansible_virt_payload.zip/ansible/modules/cloud/misc/virt.py\", line 558, in core\n  File \"/tmp/ansible_virt_payload_iiza19r9/ansible_virt_payload.zip/ansible/modules/cloud/misc/virt.py\", line 423, in status\n  File \"/tmp/ansible_virt_payload_iiza19r9/ansible_virt_payload.zip/ansible/modules/cloud/misc/virt.py\", line 251, in get_status\n  File \"/tmp/ansible_virt_payload_iiza19r9/ansible_virt_payload.zip/ansible/modules/cloud/misc/virt.py\", line 220, in find_vm\nVMNotFound: virtual machine centos71_0 not found\n", "failed": true, "invocation": {"module_args": {"name": "centos71_0", "command": "status", "uri": "qemu:///system", "state": null, "autostart": null, "xml": null, "data": "mock"}}, "changed": false, "nodeinfo": ["centos71", 0, "_", "qemu:///system"], "ansible_loop_var": "nodeinfo"}
